.github/ISSUE_TEMPLATE.md

```diff
@@ -1,28 +1,42 @@
+**PLEASE READ:** Before reporting here, *please* carefully read our **Troubleshooting Guide** at <https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ> and search the issues page here for your error/problem, as most postd bugs are duplicates or easy fixes.
```

Comment:
I think we should enclose this paragraph in a markdown comment. I'd expect a lot of people forget to remove it.

Reply:
I would otherwise do so, but my reason behind making it visible and asking the user to delete it themselves is that it serves as a canary; if it show up not-deleted on a post, its an obvious sign to both us and the user that they very likely haven't bothered to read the "Please Read" section, which states quite clearly that they should delete it, and therefore further implies that they haven't done the basic due diligence of reading the troubleshooting guide or checking for other issues first. Furthermore, making it show up on the actual post gives the user more chances to see it, both in the preview and once they've posted it. However, if you still think I should comment it, then I will of course do so; let me know.

Reply:
It seems we're split in this decision. @spyder-ide/core-developers, @spyder-ide/junior-developers, what do you think about this?

Reply:
I agree with @ccordoba12, its most likely that people will forget to remove it from their troubleshooting reports

Reply:
I would like to see a screenshot first :P

Reply:
> I agree, its most likely that people will forget to remove it from their troubleshooting reports

As mentioned above, I don't disagree that some people will.  However, my argument is that _that's the point_. If they haven't read it carefully enough to see that they should delete it once they've read it, then what I contend (which is certainly up for debate) is that this they would also very likely have either not bothered to read the lines with the troubleshooting guide, or at least not had the similar diligence to read and follow the guide itself.

Reply:
@jnsebgosselin, this text would be the first of our issue template to report new bugs here on Github. It's not part of the interface itself.

Reply:
@jnsebgosselin Just scroll up :P

Reply:
If it's a Markdown comment, then that means the user will see it, but it won't show up in the bug tracker?

Reply:
Alright thanks. Because of the VSCode screenshot, I thought it was UI related. I'm going to take a look at this and think about it later tonight.

Reply:
> If it's a Markdown comment, then that means the user will see it, but it won't show up in the bug tracker?

Yep @csabella, that's my proposal. However, @CAM-Gerlach would prefer it to leave it as a text part of the issue report.

Reply:
@ccordoba12   Thanks.  I vote for Markdown comment.    There are some people who are careful about the report they file, read everything, and would delete the comment.  But, there are others that aren't as careful (for a variety of reasons) and I don't know if blowing up the reports would really make a difference.  Yes, it may be a canary, but it's also possible they would delete it and still not have read the troubleshooting page.

However, this is also bikeshedding since we can really try it both ways and see which works better.  :-)

Reply:
@CAM-Gerlach   Can you look over your proposed changes?  There are some typos.  I wasn't sure if this was still a WIP or not, so maybe you're already taking care of it.  Thanks!

Reply:
@csabella, this is supposed to be ready to be merged, except for this part. So please point the typos so CAM can fix them.

Reply:
@csabella Thanks for going through it...no spellchecker in Spyder and while I'm generally good at finding errors in others' prose, I make plenty of mistakes myself and don't always have the detachment from my own work to notice them right away.

Reply:
I vote for enclosing in markdown. Less work for everyone.

Reply:
I would also close incomplete reports right off the bat. Such reports never get additional info anyway.

Reply:
@CAM-Gerlach, I think the issue is settled now. Please enclose the necessary text in a markdown comment, so we can merge this one ASAP.

Reply:
@ccordoba12 Yes indeed, I've done so (and a few other fixes), and final CI tests are running now.

Reply:
@jnsebgosselin I certainly wouldn't say "never", but it does seem pretty rare for those that are just a traceback and not even a descriptive title. We could just close such "driveby" issues with our usual message, and ask the OP to reopen and comment if so. However, it is only a few minutes per day to close the <kbd>status:Awaiting Followup</kbd> issues as it is, and they're easy to keep track of, so it would only save a few minutes of my time per day at most, and we should see even fewer of them once this PR and its brethren go into effect, so its really a wash IMO.